### ISSUE TITLE
Client Side Tests

### ISSUE THREAD

**Issue by sinasabet81** [RohitSat/cse5914#35]
We need some kind of testing for the client side. Preliminarily I'll be looking into Selenium for this.

**Comment by sinasabet81** [comment]
working on it in ss/selenium; going to put it on hold to integrate the sessions stuff on client-side.

**Comment by sinasabet81** [comment]
Blocked; unit tests weren't running properly, once that is working and the branch is merged we can build some fresh tests on top of it.

**Comment by sinasabet81** [comment]
Not doing due to lack of need to further maintain the application. Closing.
